rust/main/agents/scraper/src/db/message.rs

```diff
@@ -272,28 +273,28 @@ impl ScraperDb {
             })
             .collect_vec();
 
-        debug_assert!(!models.is_empty());
-        trace!(?models, "Writing messages to database");
-
-        Insert::many(models)
-            .on_conflict(
-                OnConflict::columns([
-                    message::Column::OriginMailbox,
-                    message::Column::Origin,
-                    message::Column::Nonce,
-                ])
-                .update_columns([
-                    message::Column::TimeCreated,
-                    message::Column::Destination,
-                    message::Column::Sender,
-                    message::Column::Recipient,
-                    message::Column::MsgBody,
-                    message::Column::OriginTxId,
-                ])
-                .to_owned(),
-            )
-            .exec(&self.0)
-            .await?;
+        if !models.is_empty() {
```

Comment:
here too, can `return Ok(0)` if `models.is_empty()` to avoid the new indentation

Reply:
Fixed